experiments/AtmosLES/cfsite_hadgem2-a_07_amip.jl

```diff
@@ -0,0 +1,645 @@
+# General Julia modules
+using ArgParse
+using Dierckx
+using Distributions
+using DocStringExtensions
+using LinearAlgebra
+using NCDatasets
+using Pkg.Artifacts
+using Printf
+using Random
+using StaticArrays
+using Test
+
+# ClimateMachine Modules
+using ClimateMachine
+
+using ClimateMachine.ArtifactWrappers
+using ClimateMachine.Atmos
+using ClimateMachine.ConfigTypes
+using ClimateMachine.GenericCallbacks
+using ClimateMachine.DGMethods.NumericalFluxes
+using ClimateMachine.Diagnostics
+using ClimateMachine.Mesh.Filters
+using ClimateMachine.Mesh.Grids
+using ClimateMachine.Orientations
+using ClimateMachine.Thermodynamics
+using ClimateMachine.TurbulenceClosures
+using ClimateMachine.ODESolvers
+using ClimateMachine.VariableTemplates
+using ClimateMachine.Writers
+
+# Planet parameters 
+using CLIMAParameters
+using CLIMAParameters.Planet: e_int_v0, grav, day
+struct EarthParameterSet <: AbstractEarthParameterSet end
+const param_set = EarthParameterSet()
+# Physics specific imports 
+using ClimateMachine.Atmos: altitude, recover_thermo_state
+import ClimateMachine.Atmos: source!, atmos_source!
+
+# Citation for problem setup
+
+##CMIP6 Test Dataset - cfsites
+##@Article{
+##    gmd-10-359-2017,
+##    AUTHOR = {Webb, M. J. and Andrews, T. and Bodas-Salcedo, A. and Bony, S. and Bretherton, C. S. and Chadwick, R. and Chepfer, H. and Douville, H. and Good, P. and Kay, J. E. and Klein, S. A. and Marchand, R. and Medeiros, B. and Siebesma, A. P. and Skinner, C. B. and Stevens, B. and Tselioudis, G. and Tsushima, Y. and Watanabe, M.},
+##    TITLE = {The Cloud Feedback Model Intercomparison Project (CFMIP) contribution to CMIP6},
+##    JOURNAL = {Geoscientific Model Development},
+##    VOLUME = {10},
+##    YEAR = {2017},
+##    NUMBER = {1},
+##    PAGES = {359--384},
+##    URL = {https://www.geosci-model-dev.net/10/359/2017/},
+##    DOI = {10.5194/gmd-10-359-2017},
+##}
```

Comment:
Let's move this to `docs/bibliography.bib` and cite it using `[Webb2017](@cite)`